src/explore-education-statistics-admin/src/components/editable/EditableContentBlock.tsx

```diff
@@ -32,7 +33,7 @@ const EditableContentBlock = ({
       return toHtml(value);
     }
 
-    return value;
+    return DOMPurify.sanitize(value);
```

Comment:
We should abstract this into a common string util function called `sanitizeHtml` and provide some default stricter options. It looks like DOMPurify has plenty of options we can use: https://github.com/cure53/DOMPurify#can-i-configure-dompurify

I would suggest whitelisting elements that are allowed by our editor in the admin.